typings/postcss-cssnext.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module 'postcss-cssnext' {
```

Comment:
Can we move these to `src` and just as `interfaces.d.ts`?  Being in `typings` makes it feel like it is using the old tool `typings/typings` when in fact it is directly related to the source in `src`

Reply:
Can you clarify what you mean here? Are you saying these should just be included in `interfaces.d.ts`? That doesn't seem to work since that's defining things that aren't global.

Reply:
If you don't export anything from a `.d.ts` file, it is a global/ambient definition.  If you export something from a `.d.ts` is a modular/UMD definition, then you have to deal with your globals this way:

```ts
declare global {
  // global stuff here...
}
```

Reply:
Right, but this isn't global. These type definitions are for node modules `postcss-cssnext` and `postcss-modules`.

Reply:
Ok, I still don't like `typings` because it is confusing, but I have [an example](https://github.com/nicknisi/web-editor/compare/webpack-export...kitsonk:webpack-typings) of using UMD declarations and adding extras types in a way that could be migrated to `@types` in the future...